lib/tasks/bootstrap.rake

```diff
@@ -42,14 +34,10 @@ namespace :bootstrap do
 
   desc 'Create user (optional arguments email and password)'
   task :create_user, [:email, :password] => :environment do |t, args|
-    args.with_defaults(email: 'default@loomio.com', password: 'bootstrap_password')
+    args.with_defaults(email: 'default@loomio.org', password: 'bootstrap_password')
```

Comment:
can we please use admin@example.org and generate a password which is puts'ed to the screen at the end?